go.mod

```diff
@@ -5,7 +5,9 @@ go 1.15
 require (
 	github.com/docker/go-units v0.4.0
 	github.com/jessevdk/go-flags v1.5.0
+	github.com/klauspost/compress v1.15.1
 	github.com/stretchr/testify v1.7.1
 	github.com/surma/gocpio v1.0.2-0.20130630172628-c01aebd00e02
+	github.com/ulikunitz/xz v0.5.10
```

Comment:
These dependencies are packaged in Debian too! I wonder if it would be worth doing a manual test build against the versions which are packaged in Debian before merging?

https://packages.debian.org/sid/golang-github-ulikunitz-xz-dev
https://packages.debian.org/sid/golang-github-ulikunitz-xz-dev

Reply:
Might be a good experiment. Bear it mind that I won't be able to help with that.

Reply:
Seems to build fine with those versions of the dependencies.